### ISSUE TITLE
allow groups if spaces are detected

### ISSUE THREAD

**Issue by beckend** [namnm/eslint-plugin-sort-keys#4]
example on a valid group would be:
```javascript
{
  "2": 2,
  "2-end": 2,
  
  "1": 1,
  "1-end" : 0,
}
```
